app/commands/git/temp/import_pull_requests.rb

```diff
@@ -0,0 +1,32 @@
+module Git
+  module Temp
+    class ImportPullRequests
```

Comment:
As you mentioned before, the question is: is this a temporary script? In theory, it could be. We just run at when we launch v3, use the imported PRs to update the reputation for everyone and then remove this script (and possibly the database records). From that point on, the webhook should have PR updates. I don't know how likely it is that the webhooks fails and if we will notice that. If we think we can get that (possibly failing/missing webhooks) under control (e.g. by monitoring), then this can be temporary.